src/keymap/mod.rs

```diff
@@ -18,6 +18,9 @@ pub trait KeyMap<'a, W: Write, T>: From<T> {
                 try!(self.editor().handle_newline());
                 done = true;
             }
+            Key::Ctrl('f') => {
```

Comment:
The tests break because adding `Key::Right` here prevents it from getting passed through to the underlying key bindings. If you add it here, you need to detect what the context is and forward the keypress through to the keybinding if a completion shouldn't be done (like the`_` condition below).